pyproject.toml

```diff
@@ -45,26 +45,26 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.9.0"
 cryptography = "^37.0.0"
-flask = "^2.1.0"
-flask-wtf = "^1.0.0"
+flask = "^2.0.1"
+flask-wtf = "^0.14.3"
 python-freeipa = "^1.0.6"
 click = "^8.0"
-pyjwt = "^2.3.0"
+pyjwt = "^2.1.0"
 Flask-Mail = "^0.9.1"
 gunicorn = {version = "^20.0", optional = true}
-fedora-messaging = "^3.0.0"
-backoff = "^2.0.0"
-noggin-messages = "^1.0.0"
-whitenoise = "^6.0.0"
+fedora-messaging = "^2.1.0"
+backoff = "^1.10.0"
+noggin-messages = "^0.0.1"
+whitenoise = "^5.3.0"
```

Comment:
Same here, we want to support 6.0.0, so it should be something like `">=5.3, <7.0"`